src/Microsoft.DotNet.Build.Tasks/PackageFiles/tool-runtime/project.csproj

```diff
@@ -16,14 +16,7 @@
   </ItemGroup>
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'netcoreapp2.0' ">
-    <PackageReference Include="Microsoft.Build.Framework" Version="15.3.409" />
-    <PackageReference Include="Microsoft.Build.Runtime" Version="15.3.409" />
-    <PackageReference Include="Microsoft.Build.Tasks.Core" Version="15.3.409" />
-    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="15.3.409" />
-    <PackageReference Include="Microsoft.Build" Version="15.3.409" />
-    <PackageReference Include="Microsoft.DotNet.PlatformAbstractions" Version="2.0.0" />
     <PackageReference Include="Microsoft.Net.Compilers.netcore" Version="2.6.0-beta3-62316-02" />
```

Comment:
@joperezr shouldn't Microsoft.Net.compilers.netcore also be removed now?

Reply:
Yes, we should be using `Microsoft.NETCore.Compilers which we are restoring on the init-tools.cmd and init-tools.sh now.

Reply:
So this is still needed because we are using our own custom Roslyn? If so I'm fine with keeping it until we clean that up.